tools/ovc/unit_tests/moc_tf_fe/conversion_basic_models_test.py

```diff
@@ -35,132 +35,23 @@ def basic(self, input_model, argv_input, inputs, dtype, expected, only_conversio
-                    bool,
-            ),
-        ],
-    )
-    def test_bool(self, input_freezing_value, inputs, expected,
-                  dtype=None):
-        self.basic("model_bool.pbtxt", input_freezing_value, inputs, dtype, expected)
-
-    @generate(
-        *[
-            (
-                    "in1[3]->[1 2 3],in2[3]->[4 5 6],cond->False",
-                    {},
-                    np.array([4, 5, 6], dtype=np.float32),
-                    np.float32,
-                    None
-            ),
-            (
-                    "in1,in2,cond->False",
-                    {"in1": np.array([2.0, 4.0, 6.0], dtype=np.float32),
-                     "in2": np.array([1.0, 3.0, 5.0], dtype=np.float32)},
-                    np.array([2, 4, 6], dtype=np.float32),
-                    np.float32,
-                    True  # fill a bug to investigate why compilation of this model is hang on
-            ),
-            # case: input_shape + freeze_placeholder_with_value
-            (
-                    "in2,in1->[2.0 4.0 6.0],cond->True",
-                    {"in2": np.array([1.0, 3.0, 5.0], dtype=np.float32)},
-                    np.array([2, 4, 6], dtype=np.float32),
-                    np.float32,
-                    False
-            ),
-        ],
-    )
-    def test_bool2(self, input_freezing_value, inputs, expected,
-                   dtype=None, only_conversion=False):
-        self.basic("model_bool2.pbtxt", input_freezing_value, inputs, dtype, expected,
-                   only_conversion)
-
-    @generate(
-        *[
-            (
-                    "add:0[3],z",
-                    {"add:0": np.array([4, 5, 6], dtype=np.float32), "z": np.array([1, 2, 3], dtype=np.float32)},
-                    np.array([4, 10, 18], dtype=np.float32),
-                    np.float32,
-                    None
-            ),
-            (
-                    "add:0{i32}[3],z{i32}",
-                    {"add:0": np.array([4, 5, 6], dtype=np.int32), "z": np.array([1, 2, 3], dtype=np.int32)},
-                    np.array([4, 10, 18], dtype=np.int32),
-                    np.int32,
-                    None
-            ),
-        ],
+        # TODO: Return this test when new 'cut' helper is introduced
+        # *[
+        #     (
+        #             "add:0[3],z",
+        #             {"add:0": np.array([4, 5, 6], dtype=np.float32), "z": np.array([1, 2, 3], dtype=np.float32)},
+        #             np.array([4, 10, 18], dtype=np.float32),
+        #             np.float32,
+        #             None
+        #     ),
+        #     (
+        #             "add:0{i32}[3],z{i32}",
+        #             {"add:0": np.array([4, 5, 6], dtype=np.int32), "z": np.array([1, 2, 3], dtype=np.int32)},
+        #             np.array([4, 10, 18], dtype=np.int32),
+        #             np.int32,
+        #             None
+        #     ),
+        # ],
     )
```

Comment:
please add test to check that exception about cutting is generated correctly

Reply:
Done. The test is in ovc_python_api_tests/test_complex_params.py group.